File: 백준/Silver/1929. 소수 구하기/소수 구하기.py
start, end = map(int, input().split())

for num in range(start, end + 1):
    if num == 1:
        continue
    is_prime = True
    for divisor in range(2, int(num**0.5) + 1):
        if num % divisor == 0:
            is_prime = False
            break
    if is_prime:
        print(num)

File: 백준/Silver/1929. 소수 구하기/소수 구하기.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf8').trim().split(' ');
const M = Number(input[0]);
const N = Number(input[1]);

function isPrime(num) {
  if (num < 2) return false;
  for (let i = 2; i * i <= num; i++) {
    if (num % i === 0) return false;
  }
  return true;
}

let result = [];
for (let i = M; i <= N; i++) {
  if (isPrime(i)) result.push(i);
}

console.log(result.join('\n'));